pulp_ansible/app/tasks/utils.py

```diff
@@ -43,7 +43,15 @@ def parse_metadata(download_result):
         return json.load(fd)
 
 
-RequirementsFileEntry = namedtuple("RequirementsFileEntry", ["name", "version", "source"])
+class RequirementsFileEntry(namedtuple("RequirementsFileEntry", ["name", "version", "source"])):
+    """Object to store collections from requirements file."""
+
+    def included(self, other):
+        """Simple inclusivity test, check if this RFE is part of another."""
+        if self.name == other.name:
+            if other.version == "*":
```

Comment:
It maybe doesn't matter but I figured an inequality test too? If the two versions were parsed with the semver library it could defer to their comparison for all cases other than the one written here. Just a though.

Reply:
Not sure what you mean here.  We don't compare RFE's anywhere and I'm only using this function when determining to add a dependency to the sync pipeline. It has an expectation that it is being called with a new dependency, `self.version == "*"`, to check against other RFEs already in `collection_info`.

 So say the requirement file had collection A: "*", B: "*", C: "1.0" and A depended on C: "2.0" and B depended on C: "3.0".  `collection_info` would have `[(A, "*") , (B, "*"), (C, "1.0")]` and when it synced A it would see the dependency and call this method with `(C, "*")` The method would return false and all the other versions of C would be added to the sync.  When it syncs B `collection_info` will have `[(A, "*") , (B, "*"), (C, "1.0"), (C, "*")]` and the method will return true and thus the dependency won't be added to sync again.